docs/learn/learn-parachains-protocol.md

```diff
@@ -0,0 +1,184 @@
+
+1. Parablocks that have been included by the Inclusion Pipeline are pending approval for a time
+   window known as the **secondary checking window**.
+2. During the secondary checking window, validators (secondary checkers) randomly self-select to
+   perform secondary checks on the parablock.
+3. Secondary checkers acquire the parablock with PoV and re-run the validation function.
+4. Secondary checkers gossip the results of their checks. Contradictory results lead to escalation
+   in which all validators are required to check the block. The validators on the losing side will
+   be slashed.
+5. At the end of the process the parablock is either approved or rejected.
+
+The figure below shows the path of a parachain block when it exits the Inclusion Pipeline and it
+enters the Approval Process. The parablock becomes accepted when it is backed, available and
+**undisputed**. Thr parablock is checked a second time by a subset of validators (V5, V6 and V7),
+and if there are no contradictory results the block is approved and gossiped to other relay chain
+validators. Note the parablock after second checks is shown as a square with white background a
+yellow border enclosing a "P", and three white ticks (one for each second checker). Approved
+para-blocks are shown as yellow squares the become grey.
+
+![parachain-approval-process](../assets/parachain-approval-process.png)
+
+:::info
+
+More information can be found in the dedicated sections about the
+[approval process](https://paritytech.github.io/polkadot/book/protocol-approval.html) and
+[disputes](https://paritytech.github.io/polkadot/book/protocol-disputes.html).
+
+:::
+
+## Network Asynchrony
+
+We have mentioned how a relay chain block author must select the candidate and note it on the Relay
+Chain (we say the block is backed). The relay chain block author is selected by BABE, which is a
+forkful algorithm. This means that different block authors are chosen at the same time and they may
+not work on the same block parent (i.e. the representations in the previous figures are simplistic).
+Also, the sets of validators and parachains are not fixed, and the validators' assignments to
+parachains is also flexible.
+
+We say that the network is **asynchronous** since there will be validators who have received a
+block, and other validators who did not. Thus, the network is variable and it exists in multiple
+states. In the figure below (_left_) the Group 1 received block C while Group 2 did not due to
+network asynchrony. Validators in Group 2 can build another block on top of B, called C'. Assume
+that afterwards, some validators become aware of both C and C' while others remain aware of one of
+them (_right_). Validators in Group 3 must be aware of the network state in each head (C and C') and
+they may contribute to some or full extent on both. It is possible that due to network asynchrony
+two forks may grow in parallel for some time.
```

Comment:
This section needs a conclusion. Something in the lines of "two/multiple forks may grow in parallel for some time, but eventually......"